VAMobile/src/components/LoadingComponent/LoadingComponent.tsx

```diff
@@ -13,6 +13,8 @@ export type LoadingComponentProps = {
   a11yLabel?: string
   /** Param to show the spinner icon only and not the full page with text */
   justTheSpinnerIcon?: boolean
+  /** Param to show the spinner icon only and not the full page with text */
+  noBoxSpinner?: boolean
```

Comment:
Nit, but would naming this something like `inlineSpinner` be a little clearer?

Reply:
updated